app/controllers/expenses_introduction_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class ExpensesIntroductionController < StepsController
```

Comment:
when I created this controller and the other "static" pages controllers I overrode `update` but I guess I didn't have to! *magic* ✨